src/glaciers/glacier/glacier2D_utils.jl

```diff
@@ -130,12 +129,16 @@ function initialize_glacier_data(gdir::PyObject, params::Parameters; smoothing=f
     try
         # We filter glacier borders in high elevations to avoid overflow problems
         dist_border::Matrix{F} = F.(glacier_gd.dis_from_border.data)
-        S::Matrix{F} = F.(glacier_gd.topo.data) # surface elevation
+        
             # H_mask = (dist_border .< 20.0) .&& (S .> maximum(S)*0.7)
             # H₀[H_mask] .= 0.0
 
         B::Matrix{F} = F.(glacier_gd.topo.data) .- H₀ # bedrock
         S_coords::PyObject = rioxarray.open_rasterio(gdir.get_filepath("dem"))
+        #S::Matrix{F} = F.(glacier_gd.topo.data)
+        S::Matrix{F} = F.(S_coords[1].values) # surface elevation
```

Comment:
Why has this changed?

Reply:
For some reason glacier_gd.topo.data did not change at all when trying to set different surface sources. Also OGGM tutorials pointed this as the way to extract topography data.